prow/tide/github.go

```diff
@@ -530,8 +530,8 @@ func (m *mergeChecker) isAllowedToMerge(crc *CodeReviewCommon) (string, error) {
 	}
 	mergeMethod, err := m.prMergeMethod(m.config().Tide, crc)
```

Comment:
nit: Now we don't ever use the error message contents. Since `prMergeMethod()` can only encounter one type of error this seems fine to me, but lets get rid of the error since we don't need it.  Returning `(types.PullRequestMergeType, bool)` or simply `*types.PullRequestMergeType` seems cleaner.

Reply:
Good idea, I'll address it in another PR. I'll open it soon and add you to the reviewer's list.